core/src/test/scala/com/mesosphere/usi/core/helpers/FrameworkMock.scala

```diff
@@ -0,0 +1,25 @@
+package com.mesosphere.usi.core.helpers
+
+import com.mesosphere.usi.core.matching.ScalarResourceRequirement
+import com.mesosphere.usi.core.models.Goal
+import com.mesosphere.usi.core.models.PodId
+import com.mesosphere.usi.core.models.PodSpec
+import com.mesosphere.usi.core.models.ResourceType
+import com.mesosphere.usi.core.models.RunSpec
+import com.mesosphere.usi.core.protos.ProtoBuilders
+import org.apache.mesos.v1.{Protos => Mesos}
+
+object FrameworkMock {
+
+  val mockPodId: PodId = PodId("mocked-podId")
+  val mockRunSpec: RunSpec = RunSpec(
+    List(ScalarResourceRequirement(ResourceType.CPUS, 1), ScalarResourceRequirement(ResourceType.MEM, 256)),
+    shellCommand = "sleep 3600")
+
+  val mockPodSpec: PodSpec = PodSpec(
```

Comment:
I am not a big fan of these factory methods for tests, especially if we are still exploring the API.

Also I would say that mock is something, that simulates something else (real). I would say instantiating a PodSpec is not mocking...

My vote would be to move those back to the code itself.

We had pretty huge test builders in Marathon and we're slowly moving out of those and are exercising real code in tests.

Also when using the complete definition it's easier to understand the tests when reading them.